src/OpenTelemetry.Extensions.EventSource/OpenTelemetryEventSourceLogEmitter.cs

```diff
@@ -147,9 +157,16 @@ protected override void OnEventWritten(EventWrittenEventArgs eventData)
 #if !NETFRAMEWORK
                 Timestamp = eventData.TimeStamp,
 #endif
-                Severity = ConvertEventLevelToLogLevel(eventData.Level),
             };
 
+            uint eventLevel = (uint)eventData.Level;
+            if (eventLevel < 6)
+            {
+                Tuple<LogRecordSeverity, string> eventLevelMapping = EventLevels[eventLevel];
+                data.Severity = eventLevelMapping.Item1;
```

Comment:
Does `Severity` have to be set?

Reply:
> Does Severity have to be set?

No! Pretty much everything is optional. `Timestamp` will always have a value.

> A different question: Can we just get rid of the LogRecordSeverity enum? Seems all logging frameworks should be setting SeverityText as applicable.

Let's say some logging framework out there sets SeverityText to "T-Rex" 🦖 What would a backend do with that? Seems like a good thing to encourage people to do the normalization, no?

Reply:
> 🦖

All backend MUST support emojis.

> Seems like a good thing to encourage people to do the normalization, no?

🤷 I don't personally think normalization is necessary. Though, I'm ok coming back to the question of removing Severity or not.